pkg/cmd/roachtest/tests/mixed_version_cdc.go

```diff
@@ -417,7 +417,8 @@ func (cmvt *cdcMixedVersionTester) initWorkload(
 		Flag("seed", r.Int63()).
 		Arg("{pgurl%s}", cmvt.crdbNodes)
 
-	if err := cmvt.c.RunE(ctx, option.WithNodes(option.NodeListOption{h.RandomNode(r, cmvt.workloadNodes)}), bankInit.String()); err != nil {
```

Comment:
It seems the above could also have be written as,

```
if err := cmvt.c.RunE(ctx, option.WithNodes(cmvt.workloadNodes.SeededRandNode(r)), bankInit.String()); err != nil {
```

I am a little confused by the removal of `RandNode` and the (naming) asymmetry between `SeededRandNode` and `RandomDB`.

Reply:
`RandomNode` was added before `SeededRandNode` existed. In hindsight, the former should never have existed in favour of something like the latter.

Not sure what you mean exactly re: naming asymmetry. `RandomDB` returns a _connection_ to a random node; `SeededRandNode` just picks a random node from a list.

Reply:
> Not sure what you mean exactly re: naming asymmetry. RandomDB returns a connection to a random node; SeededRandNode just picks a random node from a list.

Both are _seeded_; i.e., take prng as param., right? I'd drop the prefix `Seeded`.

Reply:
`SeededRandNode` is a function on roachtest's `cluster` that was added a few months back. I think it has that name because of the existing `RandNode` function, which uses the global RNG.

Reply:
Oh, I see; it was added by Michael. I guess we could rename it or keep as is, no strong opinion.

Reply:
I agree it could have a better name, but we should probably do that in a separate PR as it would be unrelated to the changes being introduced here.